static/app/utils/replays/replayReader.tsx

```diff
@@ -534,6 +538,35 @@ export default class ReplayReader {
     return eventsWithSnapshots;
   });
 
+  /**
+   * Filter out style mutations as they can cause perf problems especially when
+   * used in replayStepper
+   */
+  getRRWebFramesWithoutStyles = memoize(() => {
+    return this.getRRWebFrames().map(e => {
+      if (
+        e.type === EventType.IncrementalSnapshot &&
+        'source' in e.data &&
+        e.data.source === IncrementalSource.Mutation
```

Comment:
i had this same bug! forgetting about FullSnapshot.

there's a new util to help: 
```
import {isRRWebChangeFrame} from 'sentry/utils/replays/types';
```

in the other places where I've called the util i did not memoize it... because i was lazy and didn't add to ReplayReader; just chained some calls. That could probably be unified.

Reply:
In this case, we don't care about full snapshots, only incremental

Reply:
(I did copy&paste this from elsewhere though, so would be good to refactor into a helper)